public/css/common.less

```diff
@@ -27,6 +27,19 @@ pre {
   border-radius: initial;
 }
 
+.controls {
+  box-shadow: 0 0 0 1px var(--gray-lighter, #4b4b4b);
```

Comment:
Please sort and add a note that this is copied from Icinga DB Web. Also, please research the purpose of setting `flex-shrink`, `position` and `z-index` here.

Reply:
I didn't want you to comment on what each CSS property means on its own, but **why** they are set here. If all of these props are removed, nothing changes. Also, `controls` is `display: block` so setting flex props has no effect. That's why I wrote **research**.